packages/workbox-strategies/src/NetworkFirst.ts

```diff
@@ -17,7 +17,7 @@ import {messages} from './utils/messages.js';
 import './_version.js';
 
 
-interface NetworkFirstOptions extends StrategyOptions {
+export interface NetworkFirstOptions extends StrategyOptions {
```

Comment:
I believe we agreed that these shouldn't be sub-module exports rather than top-level exports, right? This matches what we're doing for the `StrategyOptions` interface as well, and means you have to do `import {NetworkFirstOptions} from 'workbox-strategies/NetworkFirst'`.

https://github.com/GoogleChrome/workbox/blob/0f52e63616888bb6b568ed8dba0ec049654529c5/packages/workbox-strategies/src/Strategy.ts#L20-L25